packages/ses/package.json

```diff
@@ -36,6 +36,8 @@
     "demo": "http-server -o /demos"
   },
   "dependencies": {
+    "@agoric/assert": "0.1.0",
+    "@agoric/console": "~0.1.0",
```

Comment:
These should both be `"^0.1.0"` once this is published. We can use version `0.0.0+1-dev` as the `"version"` in both of these for now, and make both of these dependency predicates `^0.0.0`. We bump these in the release process.

Reply:
We can't do that for `@agoric/assert`, as it is based on the `agoric-sdk`'s package which is already at `0.0.9`.  We could maybe make it `0.0.9+1-dev`.

Reply:
I don't understand the suggestion. We should discuss. Until then I made both `"^0.1.0-dev"`. Not resolving.

Reply:
Still unsure whether anything else needs to be done here. PTAL. Not resolving. Thanks.

Reply:
This PR no longer introduces new packages, so this question no longer arises. Resolving.